ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -78,7 +78,8 @@ conformance_test(
         "--exclude=ContractKeysIT,ContractKeysIT:CKFetchOrLookup,ContractKeysIT:CKNoFetchUndisclosed,ContractKeysIT:CKMaintainerScoped" +
         ",ParticipantPruningIT" +  # see "conformance-test-participant-pruning" below
         ",ConfigManagementServiceIT,LedgerConfigurationServiceIT" +  # dynamic config management not supported by Canton
-        ",ClosedWorldIT",  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
+        ",ClosedWorldIT" +  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
+        ",RaceConditionIT",  # Not yet adapted for Canton
```

Comment:
As discussed over slack, please add `RaceConditionIT` to the `conformance-test-contract-keys` to line 114 below as the suite relies on unique contract keys. - fyi @andreaslochbihler-da

Reply:
Great to see such tests coming! Is there any chance that we split the test into two pieces (one with all the tests that require unique contract keys and one for the rest)? Then we could run Canton on the latter also in the non-unique mode.

Reply:
> Great to see such tests coming! Is there any chance that we split the test into two pieces (one with all the tests that require unique contract keys and one for the rest)? Then we could run Canton on the latter also in the non-unique mode.

Good point, I was assuming all `RaceConditionIT` tests were relying on unique contract keys as the condition to race against, but that may not be the case.

Reply:
@andreaslochbihler-da I think it's a good idea!
But need @mziolekda input to decide if I should do the split in this or another PR. Personally I would opt for making it a new PR.